src/theme.cpp

```diff
@@ -422,8 +424,8 @@ theme::label::label()
 {
 }
 
-theme::label::label(const config& cfg)
-	: object(cfg)
+theme::label::label(std::size_t sw, std::size_t sh, const config& cfg)
+        : object(sw, sh, cfg)
```

Comment:
Please keep the indentation as tabs. (There are a couple more cases in this file and the header.)

Reply:
Ugh, really sorry; I though git was supposed to highlight whitespace differences when I did my final review.. I will fix this right away.

Reply:
In my other recent pull request (#4860) I was asked to squash commits fixing typos like this. Should I do that here as well?

Reply:
If in the end there'll be several meaningful commits that would be appreciated, yes. If it's just one logical commit either way the one merging can also squash.

Reply:
Ah, ok; I think this PR is pretty self-contained, so I will hold off for now and assume that the whitespace (Doh!) patch will be squashed in the merge. If there arises a need to squash, I am happy to do it now that @gfgtdf graciously showed me how.